tests/sentry/tasks/test_backfill_seer_grouping_records.py

```diff
@@ -1724,3 +1725,26 @@ def test_backfill_seer_grouping_records_manually_skip_project(self, mock_logger)
             call("backfill finished, no cohort", extra={"project_id": self.project.id}),
         ]
         assert mock_logger.info.call_args_list == expected_call_args_list
+
+    @with_feature("projects:similarity-embeddings-backfill")
+    @patch("sentry.tasks.embeddings_grouping.utils.logger")
+    @patch("sentry.tasks.embeddings_grouping.utils._make_postgres_call")
+    def test_backfill_seer_grouping_records_postgres_exception(
+        self, mock_make_postgres_call, mock_logger
+    ):
+        """
+        Test log after postgres query retries with decreased batch size
```

Comment:
Is there a reason to test the logging rather than the behavior itself?

Reply:
since the end behavior is still raising the error, testing the logging allows me to check that it was retried with the lower batch size

Reply:
Ah, I see. Up to you, but might be worth adding a comment to that effect.